docs/includes/extracts-bucket-option.yaml

```diff
@@ -1,3 +1,11 @@
+ref: bucket-option-codec
+content: |
```

Comment:
@ccho-mongodb: I removed use of `post` here and for `collection-option-codec`. There is still an existing use of it in `collection-option-collation`, but I'll fix that in a separate PR against v1.16 and attribute it to [DOCSP-37027](https://jira.mongodb.org/browse/DOCSP-37027).